app/models.py

```diff
@@ -321,6 +323,7 @@ class UpdateSessionAnswer(BaseModel):
 
     answer: Optional[answerType]
     visited: Optional[Union[bool, None]]
+    last_updated_at: datetime = Field(default_factory=datetime.utcnow)
```

Comment:
Doesn't need to be `last_updated_at` ..you can make it just `updated_at`, it's okay :)